lib/index.js

```diff
@@ -30,12 +30,8 @@ var defaults = {
       'ios > 6, android > 4.3, samsung > 3, chromeandroid > 50'
   },
   'postcss-easy-import': {
-    onImport: function(imported) {
-      // Update the watch task with the list of imported files
-      if (typeof global.watchCSS === 'function') {
-        global.watchCSS(imported);
-      }
-    }
+    transform: identity,
```

Comment:
What do you think about checking if this is a `function` instead of using `identity`?
Same with `onImport`